v5/Http/Resources/PostResource.php

```diff
@@ -123,6 +123,16 @@ private function hydrateResourceRelationships($request)
         return $result;
     }
 
+    private function getPostSource()
+    {
+        $source = "web";
+        $message = $this->message;
+        if ($message) {
```

Comment:
Minor change to prevent redundant control flow.

```suggestion
        return $message && isset($message->type)
          : $message->type
          : self::DEFAULT_SOURCE_TYPE;
```